### DISCUSSION TITLE
使用Gcore CDN后视频在VPS下载后才播放/视频无限加载

### DISCUSSION THREAD

**Discussion by skyrim663** [AlistGo/alist#2143] [Q&A]
两台VPS使用同一份配置,nginx配置
location / {
  proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
  proxy_set_header Host $http_host;
  proxy_set_header X-Real-IP $remote_addr;
  proxy_set_header Range $http_range;
	proxy_set_header If-Range $http_if_range;
  proxy_redirect off;
  proxy_pass http://127.0.0.1:5244;
  # the max size of file to upload
  client_max_body_size 20000m;
}

使用Gcore cdn 后网页打开同一个视频，Gcore cdn的网页无线加载，直连的网页正常播放，且退出后套了Gcore的VPS后台满速下载无法自动停止，另一台正常。

之前直连的套cdn，后也会有同样的问题

**Comment by rakaki** [comment]
请教下怎么使用Gcore？

**Comment by YouHaveGone** [comment]
兄弟你GCORE是怎么设置的，我把域名托管到GCORE 和只用GCORE的cdn然后在cf上做CNAME解析都没法用。alist是能打开但都是没办法登录进到后台管理

**Reply by X1769803359** [reply]
关闭cdn缓存应该就可以了， 我gcore设置的是Origin pull protocol选https，CDN caching选关闭，SSL
开 等获取ssl证书成功就可以了
